### PR TITLE
Add LICENSE file.

### PR DESCRIPTION
**Pull request by mkillianey**

### CODE REVIEW

**Comment by mkillianey** [comment]
How would you like to be attributed for licensing purposes? I'm guessing that since you published under the MIT license, you wanted to use that template, but if this isn't how you want your attribution to appear, could you provide the full text that you'd like to be used?

**Comment by mkillianey** [comment]
@einaros -- can you verify that this is how you'd like to be attributed for your work?

**Comment by mkillianey** [comment]
@3rd-Eden -- given that the latest commit on master is from you, are you an owner?  If so, could you please let me know if this is a satisfactory way to attribute the authors' work?

File: README.md [hunk 1]
```diff
@@ -56,7 +56,3 @@ Remove a WebSocket mask on the given data.;w
 ```js
 bu.unmask(buffer, mask);
 ```
-
-## License
-
-MIT
```
Comment (lpinca) [inline README.md]: Instead or removing it from the README I would add a link to the LICENSE file (`[MIT](LICENSE)`).

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2013-2015 Einar Otto Stangvik <einaros@gmail.com> (http://2x.io)
```
Comment (lpinca) [inline LICENSE]: Can you please update the years or only keep the first? Thanks.

**Comment by lpinca** [comment]
Merged as b7da2c2f7a95826f22caf004b092dc7a65877e86.
Thank you.

**Pull request closed without merging**